Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -331,6 +333,7 @@ open class TopBannerViewController: UIViewController {
             
             if !self.isDisplayingPreviewInstructions {
                 self.showSecondaryChildren(completion: complete)
```

Comment:
Not only `LanesView` can be affected by this issue, but also `NextBannerView`, `JunctionView` etc. What do you think about renaming `showSecondaryChildren` to `showSecondaryChildrenIfNeeded` and re-implementing its logic to use `currentInstruction` member variable to see if there are any instructions that contain data for `LanesView` and other views. If data is present we'll show those views, if not they'll remain invisible as all of them were hidden when presenting steps table.

Reply:
Wouldn't this renaming break backwards-compatibility?
Using the `currentInstruction` to see what needs to be displayed in a sensible approach. I'll work on re-implementing this logic.

Reply:
I think it should be fine as `showSecondaryChildren(completion:)` is a private function.